### PR TITLE
Split battle_anim_80FE840 and move data to C

### PR DESCRIPTION
**Pull request by garakmon**

### CODE REVIEW

**Comment by garakmon** [comment]
any thoughts on this? (might be easier to look at after #372 is merged)

**Comment by garakmon** [comment]
OK this is done

File: ld_script.txt [hunk 1]
```diff
@@ -459,7 +476,26 @@ SECTIONS {
         src/field_special_scene.o(.rodata);
         src/rotating_gate.o(.rodata);
         src/item_use.o(.rodata);
-        data/battle_anim_80FE840.o(.rodata);
+        /*src/battle/anim/effects_1.o(.rodata);*/
```
Comment (Diegoisawesome) [inline ld_script.txt]:
```suggestion

```
Remove this line

File: src/effects_1.c [hunk 2]
```diff
@@ -0,0 +1,2093 @@
+#include "global.h"
```
Comment (Diegoisawesome) [inline src/effects_1.c]: Will these files eventually contain code? If not they should go into src/data

Comment (garakmon) [reply src/effects_1.c]: yes they will

File: ld_script.txt (hunk 1 above)
Comment (garakmon) [reply ld_script.txt]: oops my bad

**Pull request merged**